frontend/src/metabase/visualizations/visualizations/PieChart.jsx

```diff
@@ -171,7 +171,7 @@ export default class PieChart extends Component<*, Props, *> {
         let value, title;
         if (hovered && hovered.index != null && slices[hovered.index] !== otherSlice) {
             title = slices[hovered.index].key;
```

Comment:
Should we do `formatDimension` for the `title`? Try grouping by a date/time and see if it's formatted nicely or as a raw timestamp.